frontend/src/views/LoginPage/LoginPage.scss

```diff
@@ -0,0 +1,33 @@
+.login-page {
+  display: flex;
+  justify-content: space-around;
+}
+
+.login-container {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.login-form {
+  margin-top: 40px;
+  align-self: stretch;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  gap: 20px;
+}
+
+.login-page img {
+  height: 100vh;
+  width: auto;
+}
+
+@media screen and (max-width: 800px) {
+  .login-page img {
+    display: none;
+  }
+}
```

Comment:
it may be good idea to have those selectors more specific - nest login-container, login-form etc. inside login-page